giving-a-presentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Giving a Presentation
+sidebar_link: false
+---
+
+Boston Python's presentation events are a place for members of our community
+to share what they've learned about making python programs. Giving a
+presentation to our audience is a way to give back to the community, and it
+helps you understand the subject matter better - it's a win all around!
+
+Presenters do not need to be "experts", they do not need to have contributed to
+the python language, created a library, or written a book. Anyone who has worked
+with the language and learned something interesting from doing so has something
+to share, and our community wants to hear you share it.
+
+### How do I know if I have something to say?
+If you're not sure whether you've got a Boston Python presentation in you, take
+this quiz:
+
+- Have you learned something about the python language?
+- Has it been useful to you?
+- Do you think someone else might find it useful?
+
+If you answered "yes" to any of those questions, there's a presentation waiting
+to happen - and we'll help you find it, if you want.
+
+### What sorts of talks are you interested in?
+We're open to topics of interest to a community of python users and learners,
+ranging from quick "lightning talks" to deep-dive talks. Ideally, talks should
+be pitched to an audience including users of all levels, from beginner to
+experienced professional.
+
+### But really, what should I talk about?
+If you think a topic would be interesting to you, that's a good start! To
+inspire you, we have a long list of [past talks](https://about.bostonpython.com/past-events/)
+If you're still not sure, reach out to an organizer and we'll help you find a topic.
+
+### Okay, I'm interested. Where do I start?
+If you're interested in presenting to the Boston Python community, and you have
+a topic in mind, you can start by filling out
+[this form](https://docs.google.com/forms/d/1ndedcjLoPiq6k_A7Q1eyxHk_8rMKBC-m2CZ6Pg9NlvE/edit).
+If you have questions about giving a presentation, you can always reach out to
+an organizer on the Boston Python [slack channel](slack.md) (Jon, Ned, and
```

Comment:
```suggestion
an organizer in the Boston Python [Slack workspace](slack.md) (Jon, Ned, and
```
But we should also indicate the #organizing channel.

Reply:
We could also say something here about our willingness to help with any phase of the preparation: ideation, drafting, polishing, rehearsing, etc.